scraper/scraper.py

```diff
@@ -175,7 +179,7 @@ def get_address(self, place_feed):
 
     def get_phone_number(self, place_feed):
         element = self.get_element(place_feed,
-                                   ".UaQhfb > :nth-child(4) > :nth-child(4) > :nth-child(2) > jsl > :nth-child(2)")
+                                   ".UaQhfb > .W4Efsd > :nth-child(3) > :nth-child(2) > jsl > :nth-child(2)")
```

Comment:
If you use `.W4Efsd`, I'm not sure which of the below elements the selector will match because they both have the same class.

<img width="520" alt="Screenshot 2023-02-16 at 01 32 26" src="https://user-images.githubusercontent.com/951656/219124258-94720304-6e2f-423b-891c-b851cedfb3bd.png">

I think the correct selector should be 

```
.UaQhfb > :nth-child(4) > :nth-child(3) > :nth-child(2) > jsl > :nth-child(2)
```

Reply:
Honestly, I have no idea how my code worked at that time! And I think it shouldn't work. Anyway, this problem can be solved better